src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java

```diff
@@ -1044,7 +1044,8 @@ private void findBombardOptions(final PlayerID player, final List<Territory> myU
       // Check strafing and using allied attack if enemy capital/factory
       boolean isEnemyCapitalOrFactory = false;
       final TerritoryAttachment ta = TerritoryAttachment.get(t);
-      if ((ta != null && ta.isCapital()) || ProMatches.territoryHasInfraFactoryAndIsLand(player).match(t)) {
+      if (!t.getOwner().isNull()
```

Comment:
This seems like the wrong place for the null check. Seems like the NPE bug is in `ProMatches.territoryHasInfraFactoryAndIsLand()`, and not here.